.idea/codeStyles/Project.xml

```diff
@@ -4,6 +4,7 @@
     <option name="RIGHT_MARGIN" value="140" />
     <option name="SOFT_MARGINS" value="140" />
     <JetCodeStyleSettings>
+      <option name="LINE_BREAK_AFTER_MULTILINE_WHEN_ENTRY" value="false" />
```

Comment:
after updating IDEA, noticed this code style changed on default, so adding it here to persist config